doc/Makefile

```
@@ -7,29 +7,19 @@ SPHINXAUTOGEN = sphinx-autogen
 BUILDDIR      = _build
 
 # Internal variables.
-ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
+ALLSPHINXOPTS = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
 
 .PHONY: help clean html linkcheck doctest api
 
-all: html
-
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
 	@echo "  all        generate the complete webpage"
-	@echo "  html       make only the HTML files from the existing rst sources"
-	@echo "  linkcheck  check all external links for integrity"
-	@echo "  doctest    run all doctests embedded in the documentation (if enabled)"
+	@echo "  html       build the HTML files from the existing rst sources"
+	@echo "  api        generate API documentation rst sources"
```

```suggestion
	@echo "  api        generate API documentation from rst sources"
```

---

I find `make api` just generates rst files. It is `make html` that generate the documentation or website, right?

---

Ah you're right, I just thought the sentence looked funny. In that case maybe do something like this?

```suggestion
	@echo "  api        generate rst source files of API documentation"
```